client/reader.js

```diff
@@ -106,37 +107,50 @@ rl.on('line', (line) => {
   // if too much junk data, make sure '-p' option isn't enabled in multimon
   if (address.length > 2 && message) {
     var padAddress = padDigits(address,7);
-  	console.log(colors.red(time+': ')+colors.yellow(padAddress+': ')+colors.success(trimMessage));
-  	// now send the message
-  	var options = {
-  		method: 'POST',
-  		uri: uri,
-  		headers: {
-  			'X-Requested-With': 'XMLHttpRequest',
-  			apikey: apikey
-  		},
-  		form: {
-  			address: padAddress,
-  			message: trimMessage,
-  			datetime: datetime,
-  			source: identifier
-  		}
-  	};
-  	rp(options)
-  	    .then(function (body) {
-  	    //    console.log(colors.success('Message delivered. ID: '+body)); 
-  	    })
-  	    .catch(function (err) {
-  	        console.log(colors.yellow('Message failed to deliver. '+err));
-  	    });
+    console.log(colors.red(time+': ')+colors.yellow(padAddress+': ')+colors.success(trimMessage));
+    // now send the message
+    var form = {
+      address: padAddress,
+      message: trimMessage,
+      datetime: datetime,
+      source: identifier
+    };
+    sendPage(form, 0);
   } else {
-  	console.log(colors.red(time+': ')+colors.grey(line));
+    console.log(colors.red(time+': ')+colors.grey(line));
   }
   
 }).on('close', () => {
   console.log('Input died!');
 });
 
+var sendPage = function(message,retries) {
+  var options = {
+    method: 'POST',
+    uri: uri,
+    headers: {
+      'X-Requested-With': 'XMLHttpRequest',
```

Comment:
Can we add 
`'User-Agent': "reader.js",

Reply:
This will make filtering out API requests from decoders a simple task

Reply:
Makes sense... will that mess with CloudFlare at all? Haven't checked what user-agent it sends at the moment

Reply:
Looks like it's only an issue if you manually enable UA filtering so should be all good

Reply:
Added as "PagerMon reader.js" :)